docs/src/user-guide/quick-start/clp-text.md

```diff
@@ -0,0 +1,158 @@
+# clp-text quick-start
+
+This page will walk you through how to start CLP and use it to compress and search unstructured
+text logs.
+
+:::{caution}
+If you're using a `clp-text` release, you should only compress unstructured text logs. `clp-text`
+is able to compress and search JSON logs as if they were unstructured text, but `clp-text` cannot
+query individual fields. This limitation will be addressed in a future version of CLP.
+:::
+
+---
+
+## Starting CLP
+
+To start CLP, run:
+
+```bash
+sbin/start-clp.sh
+```
+
+:::{note}
+If CLP fails to start (e.g., due to a port conflict), try adjusting the settings in
+`etc/clp-config.yml` and then run the start command again.
+:::
+
+---
+
+## Compressing unstructured text logs
+
+To compress some unstructured text logs, run:
+
+```bash
+sbin/compress.sh <path1> [<path2> ...]
+```
+
+`<path...>` are paths to unstructured text log files or directories containing such files.
+
+The compression script will output the compression ratio of each dataset you compress, or you can
+use the UI to view overall statistics.
+
+Compressed logs will be stored in the directory specified by the `archive_output.storage.directory`
+config option in `etc/clp-config.yml` (`archive_output.storage.directory` defaults to
+`var/data/archives`).
+
+### Sample logs
+
+For some sample logs, check out the [open-source datasets](../resources-datasets).
+
+---
+
+## Searching unstructured text logs
+
+You can search your compressed logs from CLP's [UI](#searching-from-the-ui) or the
+[command line](#searching-from-the-command-line).
+
+In clp-text, queries are written as wildcard expressions. A wildcard expression is a plain text
+query where:
+
+* `*` matches zero or more characters
+* `?` matches any single character
+
+For example, consider the query in [Figure 1](#figure-1) and the logs in [Figure 2](#figure-2).
+
+(figure-1)=
+:::{card}
+
+```bash
+"INFO container_? Transitioned*ACQUIRED"
+```
+
++++
+**Figure 1**: An example query.
+:::
+
+(figure-2)=
+:::{card}
+
+```text
+2015-03-23T15:50:17.926Z INFO container_1 Transitioned from ALLOCATED to ACQUIRED
+2015-03-23T15:50:17.927Z ERROR Scheduler: Error trying to assign container token
+java.lang.IllegalArgumentException: java.net.UnknownHostException: i-e5d112ea
+    at org.apache.hadoop.security.buildTokenService(SecurityUtil.java:374)
+    at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2033)
+Caused by: java.net.UnknownHostException: i-e5d112ea
+    ... 17 more
+```
+
++++
+**Figure 2**: A set of unstructured text log events.
+:::
+
+The query in [Figure 2](#figure-2) will match with the first log message, as the `?` will stand in
+for any one character (in this case, `1`), and the `*` will stand in for the characters
+` from ALLOCATED to `.
```

Comment:
```suggestion
The query in [Figure 2](#figure-2) will match with the first log message, as the `?` will match the
character "1", and the `*` match the text " from ALLOCATED to ".
```

I'm using quotes rather than backticks since the spaces surrounding the text only get rendered if we use quotes.